### PR TITLE
[zh] add content/zh/news/releases/1.15.x/change-notes/index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
@yanrongshi I do not know. It seems it doesn't like the publishdate when I looked earlier. It's a holiday here today, but I'll try and pull your PR tomorrow and see if I can identify the problem.

**Comment by ericvn** [comment]
The other idea I have is maybe it's not a `-` in your date, but another character that looks like it and maybe that would fail the spellcheck.

**Comment by ericvn** [comment]
Running `make lint` locally yields `publishdate` as misspelled. Looking some more.

File: content/zh/news/releases/1.15.x/change-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+ ---
```
Comment (ericvn) [inline content/zh/news/releases/1.15.x/change-notes/index.md:1]: The issue is you have a leading whitespace on this line so it ends up spell checking this section.
```suggestion
---
```

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**